### PR TITLE
Refactoring of routers

### LINKED ISSUE #246: Route name refactoring
Currently all the routes and their names are defined in `RouteGenerator`. So for navigation I need to copy over the same route string. Where there is code duplication there is a chance of bugs. So to get rid of this situation i planned this refactoring task. 

We will create a static route string in the same screen widget and will access this variable in `RouteGenerator` and `Navigator`.
pseudo code
```dart
Class ScreenName extends StatelessWidget {
  static String routeName = '/route';
}
```
use
```dart
 case ScreenName .routeName:
        return MaterialWithModalsPageRoute(
          builder: (context) => ScreenName (),
        );
```
```dart
Navigator.of(context).pushNamed(
                      PackageSheet.routeName,
)
```

### PR DESCRIPTION
**Pull request by samrath-sudesh-acharya**
I have refactored the routes as you wanted ....
:)
closes #246

### CODE REVIEW

File: lib/screens/acknowledgement_sheet/acknowledgement_sheet.dart [hunk 1]
```diff
@@ -11,6 +11,7 @@ import '../package_sheet.dart';
 part 'license_data.dart';
 
 class AcknowledgementSheet extends StatelessWidget {
+  static const String Acknowledgement = '/preference/acknowledgement';
```
Comment (hashirshoaeb) [inline lib/screens/acknowledgement_sheet/acknowledgement_sheet.dart:14]: Can you name all the static variables same as 'route'. 

```diff
- static const String Acknowledgement = '/preference/acknowledgement';
+ static const String route = '/preference/acknowledgement';

File: lib/screens/activity_edit_sheet.dart [hunk 2]
```diff
@@ -15,6 +17,7 @@ import '../widgets/my_container.dart';
 ///
 /// Input form to create update and delete [Activity]
 class ActivityEditSheet extends StatefulWidget {
+  static const String Title = '/edit/title';
```
Comment (hashirshoaeb) [inline lib/screens/activity_edit_sheet.dart:20]: Similarly rename the variable.

```diff
- static const String Title = '/edit/title';
+ static const String route = '/edit/title';
```

**Review by hashirshoaeb — CHANGES_REQUESTED** [review]
Thanks for the pull request. Nice work. 🚀  Just need a small change. 

Change the static route variable names to `route`. This way we will have consistent on route naming. 

For Example.

PreferenceScreen.route
LicenseScreen.route 

Got my point?

File: lib/screens/acknowledgement_sheet/acknowledgement_sheet.dart (hunk 1 above)
Comment (samrath-sudesh-acharya) [reply lib/screens/acknowledgement_sheet/acknowledgement_sheet.dart:14]: ok got it

File: lib/screens/activity_edit_sheet.dart (hunk 2 above)
Comment (samrath-sudesh-acharya) [reply lib/screens/activity_edit_sheet.dart:20]: ok

**Comment by hashirshoaeb** [comment]
Thanks 🚀

**Pull request merged**